src/auth/auth.services.ts

```diff
@@ -7,7 +7,7 @@ export type DecodedToken = {
 
 const prisma = new PrismaClient();
 const SECRET = process.env.SECRET_KEY as string;
```

Comment:
Consider all env variables should be imported as a constant resource through all the project.

Reply:
Resolved, created a constant folder with all the variables, and exported them